include/lib/aarch64/arch.h

```diff
@@ -647,7 +647,7 @@
 #define PAR_F_SHIFT	0
 #define PAR_F_MASK	1
 #define PAR_ADDR_SHIFT	12
-#define PAR_ADDR_MASK	(BIT(40) - 1) /* 40-bits-wide page address */
+#define PAR_ADDR_MASK	(BIT(52) - ULL(1))
```

Comment:
nit: Would be good to make comment similar to the commit message here.

Reply:
Done.